### DISCUSSION TITLE
早上跑更新後就開不了 請問該怎麼解決

### DISCUSSION THREAD

**Discussion by mikelr101** [MaaAssistantArknights/MaaAssistantArknights#7608] [Q&A]
MAA 遇到了问题
設定屬性 'System.Windows.ResourceDictionary.DeferrableContent' 時擲回例外狀況。
已經加入項目。字典中的索引鍵: 'TrainingCompleted'  加入的索引鍵: 'TrainingCompleted'
解决方案
出现未知错误
详细信息
System.Windows.Markup.XamlParseException: 設定屬性 'System.Windows.ResourceDictionary.DeferrableContent' 時擲回例外狀況。 ---> System.ArgumentException: 已經加入項目。字典中的索引鍵: 'TrainingCompleted'  加入的索引鍵: 'TrainingCompleted'
   於 System.Collections.Hashtable.Insert(Object key, Object nvalue, Boolean add)
   於 System.Windows.ResourceDictionary.SetKeys(IList`1 keyCollection, IServiceProvider serviceProvider)
   於 System.Windows.Baml2006.WpfSharedBamlSchemaContext.<>c.<Create_BamlProperty_ResourceDictionary_DeferrableContent>b__299_0(Object target, Object value)
   於 MS.Internal.Xaml.Runtime.ClrObjectRuntime.SetValue(Object inst, XamlMember property, Object value)
   --- 內部例外狀況堆疊追蹤的結尾 ---
   於 System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   於 System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   於 System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   於 System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   於 System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   於 MS.Internal.AppModel.AppModelKnownContentFactory.BamlConverterCore(Stream stream, Uri baseUri, Boolean canUseTopLevelBrowser, Boolean sandboxExternalContent, Boolean allowAsync, Boolean isJournalNavigation, XamlReader& asyncObjectConverter, Boolean isUnsafe)
   於 MS.Internal.AppModel.MimeObjectFactory.GetObjectAndCloseStreamCore(Stream s, ContentType contentType, Uri baseUri, Boolean canUseTopLevelBrowser, Boolean sandboxExternalContent, Boolean allowAsync, Boolean isJournalNavigation, XamlReader& asyncObjectConverter, Boolean isUnsafe)
   於 System.Windows.ResourceDictionary.set_Source(Uri value)
   於 MaaWpfGui.Helper.LocalizationHelper.Load()
   於 MaaWpfGui.Main.Bootstrapper.OnStart()
   於 Stylet.BootstrapperBase.Start(String[] args)
   於 System.Windows.Application.OnStartup(StartupEventArgs e)
   於 MaaWpfGui.App.OnStartup(StartupEventArgs e)
   於 System.Windows.Application.<.ctor>b__1_0(Object unused)
   於 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   於 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by EROTCZ** [comment]
我也遇到這問題 回報了    目前先用著v4.28.0版本

**Comment by status102** [comment]
Duplicate of #7599
